os/tools/mksmartfsimg.sh

```diff
@@ -21,12 +21,18 @@
 
 source .config
 
-BOARDNAME=$CONFIG_ARCH_BOARD
+BOARDNAME=artik05x
```

Comment:
This script should not be board-specific. This is not acceptable.

Reply:
Yes, I will revert back.